lib/Runtime/Library/ConcatString.h

```diff
@@ -9,18 +9,30 @@ namespace Js
     class LiteralStringWithPropertyStringPtr : public LiteralString
     {
     private:
-        PropertyString * propertyString;
+        // TODO: Can we make PropertyString && LiteralStringWithPropertyStringPtr share the string buffer?
```

Comment:
I think you can just overwrite the buffer on this once we populate the PropertyString field (and then PropertyString, LiteralStringWithPropertyStringPtr, and PropertyRecord will all be sharing same buffer)

Reply:
Indeed, this is a trivial TODO. However, I would like to keep it separate. (That's why TODO) BTW; the issue is still open and it also mentions this improvement.

Reply:
done.